packages/flutter/lib/src/material/dialog.dart

```diff
@@ -186,9 +186,10 @@ class Dialog extends StatelessWidget {
   /// See the enum [Clip] for details of all possible options and their common
   /// use cases.
   ///
-  /// Defaults to [Clip.none].
+  /// If null, then [DialogTheme.clipBehavior] is used. If that is also null,
+  /// the defaults to [Clip.none].
```

Comment:
```suggestion
  /// defaults to [Clip.none].
```

Reply:
Thanks for the review! I committed it in [Update packages/flutter/lib/src/material/dialog.dart](https://github.com/flutter/flutter/pull/147635/commits/4af870090549fe21a353f731df6a56a563ab8808)

Reply:
Oh no! now the pipeline is failing because of your CLA @bleroux

Reply:
> Oh no! now the pipeline is failing because of your CLA @bleroux

@ValentinVignal Oh, sorry for that, I just noticed that the GitHub web operations use the primary email address set un my account which is not the one I use for filing PR. I have updated it but this won't change this last commit on your PR.
So the solution I see is that you delete this last commit, create a similar one and force push. Really sorry for that 😅

Reply:
Okay sure, I'll do that :)

Reply:
It should be good to go now 👍